### ISSUE TITLE
Accessibility regression in About screen

### ISSUE THREAD

**Issue by TheLastProject** [CatimaLoyalty/Android#2907]
The new Compose about screen says "Greater than arrow" on each row, which was correctly hidden from Talkback in the old UI.

This is not a major issue, but a slight annoyance/confusion risk for users with accessibility needs so this should be fixed by correctly ignoring the element from Talkback again.
